src/c++/perf_analyzer/request_rate_manager.cc

```diff
@@ -99,6 +100,7 @@ cb::Error
 RequestRateManager::ResetWorkers()
 {
   PauseWorkers();
+  ConfigureThreads();
```

Comment:
I don't think this is needed. In fact, do you see any call to ResetWorkers anywhere? Or is this deadcode?

Reply:
This appears to be deadcode. You can remove this line, and I will follow up about ripping out this code and associated testing

Reply:
Will be cleaned up here:
https://github.com/triton-inference-server/client/pull/304

You will have conflict unless you remove this line.